run-tests

```diff
@@ -139,3 +139,33 @@ if [[ $fail -eq 1 ]] ; then
 else
 	echo -e "$MSG_PASSED"
 fi
+
+
+if [[ $EXPECT_5LVL -eq 1 ]] ; then
+	echo "Running mmap+memset+fork test for 5-level page table"
+	if [[ $EXPECT_5LVL -eq 1 ]] ; then
+		BEGIN="--begin 47"
+		END="--end 56"
+	else
+		BEGIN="--begin 37"
+		END="--end 46"
+
+	fi
+
+	output="$(./mmap+memset+fork $BEGIN $END)"
```

Comment:
In the [Travis](https://travis-ci.com/sanskriti-s/pg-table_tests/builds/79977745) build,  `make check` complains about:
```
output="$(./mmap+memset+fork $BEGIN $END)"
                             ^-- SC2086: Double quote to prevent globbing and word splitting.
                                    ^-- SC2086: Double quote to prevent globbing and word splitting.
```
The ShellCheck linter  is [requesting](https://github.com/koalaman/shellcheck/wiki/SC2086) that `$BEGIN` and `$END` to be double-quoted, like:
```
output="$(./mmap+memset+fork "$BEGIN" "$END")"
```
This is generally considered good shell scripting style.  Consider this example:
```
ARG="a bunch of text"

./foo $ARG
foo:  argv[0]=foo, argv[1]=a, argv[2]=bunch, argv[3]=of, argv[4]=text

./foo "$ARG"
foo:  argv[0]=foo, argv[1]=a bunch of text
```
See the difference?  

One more thing -- we can't just blindly apply ShellCheck's suggestion in our case.  Why?  If we did, we would end up with an argument like "--begin 47"...  we actually _do_ want two arguments "--begin" and "47" (as if we left it unquoted in the first place).  My suggestion to keep good style and appease ShellCheck would be to use this form:
```
BEGIN="47"
END="56"
...
output="$(./mmap+memset+fork ... --begin "$BEGIN" --end "$END")"
```